contrib/python/podman/libs/containers.py

```diff
@@ -93,6 +94,45 @@ def export(self, target):
             results = podman.ExportContainer(self.id, target)
         return results['tarfile']
 
+    def commit(self,
+               image_name,
+               *args,
+               changes=[],
```

Comment:
Some linters will complain about a a mutable default like this.  Can we use a tuple instead?  `changes=()`

Reply:
@wking Reference, please.  None of the linters I've tried complain on this.  Varlink has shown itself to be a little brittle when passing in unexpected data types so I'd rather not change to a tuple. 

I'm interested in what the issue is with the mutable type who's life is just the duration of the method.